forge-kernel/src/tools/forge_verify.cpp

```
@@ -0,0 +1,744 @@
+        switch (s[++i]) {
+            case 'n': out.push_back('\n'); break;
+            case 't': out.push_back('\t'); break;
+            case 'r': out.push_back('\r'); break;
+            case 'b': out.push_back('\b'); break;
+            case 'f': out.push_back('\f'); break;
+            case '"': out.push_back('"'); break;
+            case '\\': out.push_back('\\'); break;
+            case '/': out.push_back('/'); break;
+            case 'u': {
+                if (i + 4 < s.size()) {
+                    const int cp = std::stoi(s.substr(i + 1, 4), nullptr, 16);
```

Clang (14.0.6)

[warning] variable name 'cp' is too short, expected at least 3 characters

(readability-identifier-length)